template/package.json

```diff
@@ -8,9 +8,10 @@
     "clean": "rimraf build",
     "lint": "eslint source test",
     "prebuild": "npm run clean",
-    "build": "NODE_ENV=production webpack && npm run build:min && npm run build:doc",
-    "build:min": "NODE_ENV=production MINIFY=1 webpack",
-    "build:doc": "doctoc --github --title '## Contents' ./",
+    "build": "npm run build:webpack && npm run build:min && npm run build:doc",
+    "build:webpack": "node scripts/buildWebpack.js NODE_ENV=production",
```

Comment:
I think we should just use `./scripts/buildProduction.js`, `./scripts/buildMinified.js` and so on. Not using ENV vars.